doc/source/dev/dev_quickstart.rst

```diff
@@ -0,0 +1,104 @@
+.. _dev-quickstart:
+
+============================
+Contributor quickstart guide
+============================
+
+After :ref:`getting the source code from GitHub <git-start>`, there are three
+steps to start contributing:
+
+1. **Set up a development environment**
+   
+   Using ``conda``, or some flavor of the many virtual environment management
+   tools, you can make sure the development version of SciPy does not interfere
+   with any other local installations of SciPy on your machine.
+
+2. **Build SciPy**
+   
+   SciPy uses compiled code for speed, which means you might need extra
+   dependencies to complete this step depending on your system.
+
+3. **Perform development tasks**
+   
+   These can include any changes you want to make to the source code, running
+   tests, building the documentation, running benchmarks, etc.
+
+Basic workflow
+==============
+
+.. note::
+
+    We **strongly** recommend using a virtual environment setup, such as
+    ``venv`` or ``conda``.
+
+Since SciPy contains parts written in C, C++, and Fortran that need to be
+compiled before use, make sure you have the necessary compilers and Python
+development headers installed. If you are using ``conda``, these will be
+installed automatically. If you are using ``pip``, check which
+:ref:`system-level dependencies <system-level>` you might need.
+
+First, fork a copy of the main SciPy repository in GitHub onto your own
+account and then create your local repository via::
+
+    git clone git@github.com:YOURUSERNAME/scipy.git scipy
+    cd scipy
+    git submodule update --init
+    git remote add upstream https://github.com/scipy/scipy.git
+
+Next, set up your development environment.
+
+.. tabs::
+
+    .. tab:: pip+venv
+
+        With :ref:`system-level dependencies <system-level>` installed, execute
```

Comment:
People are going to miss this easily, it's a few words and the contrast of the link isn't great (at least on my macbook screen). How about making "With ... installed" bold?